src/main/java/com/team01/carsharingapp/dto/payment/PaymentRequestDto.java

```diff
@@ -0,0 +1,13 @@
+package com.team01.carsharingapp.dto.payment;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Size;
+
+public record PaymentRequestDto(@NotBlank
```

Comment:
Let's check for negative ids here
Also, add message for exception
```suggestion
public record PaymentRequestDto(@NotBlank(message = "id can't be null")
@Min(value = 0, message = "id can't be less than 0")
```